Package.swift

```diff
@@ -0,0 +1,23 @@
+// swift-tools-version: 5.8
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "NetworkService",
```

Comment:
We use too much "NetworkService" strings here.
Can we avoid extract them into the meaningful constants to avoid accidental typos?

Reply:
fixed